README.md

```diff
@@ -6,6 +6,25 @@ __MS-AMP__ is an automatic mixed precision package for deep learning developed b
 
 ## _Check [aka.ms/msamp/doc](https://aka.ms/msamp/doc) for more details._
 
+## Publication
```

Comment:
Seems there is lint issue for this PR. Use below commands to check the lint errors.
```bash
sudo apt-get update
sudo apt-get install rubygems
sudo gem install mdl
make mdlint
```

Reply:
Thanks Yuxiang! I have fixed it.